flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParser.java

```diff
@@ -218,7 +217,11 @@ private List<Operation> processCmd(
             // parse statement to get AST
             final HiveParserASTNode node = HiveASTParseUtils.parse(cmd, context);
             Operation operation;
-            if (DDL_NODES.contains(node.getType())) {
+            if (node.getType() == HiveASTParser.TOK_RELOADFUNCTION) {
+                // call  reload function
+                hiveCatalog.reloadFunctions();
```

Comment:
Calling HiveCatalog#reloadFunctions directly in Parser looks very hack.
If it is as said in JIRA, is there no need to call it?

Reply:
Good catch. It's the first version for I mistake it that we need to call  reload functions.   But after some investigation about the function load mechanism in Flink, I found we don't need to reloadFunctions. It should be removed. But I haven't removed it